content/api/commands/origin.md

```diff
@@ -507,27 +507,37 @@ into the callback.
 
 ### Dependencies / Sharing Code
 
-It is not possible to use
+[ES module dynamic `import()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports)
+and/or
 [CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
-or
-[dynamic ES module `import()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports)
-within the callback. However, [`Cypress.require()`](/api/cypress-api/require)
-can be utilized to include [npm](https://www.npmjs.com/) packages and other
-files. It is functionally the same as using
-[CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
-in browser-targeted code.
+can be used within the callback to include [npm](https://www.npmjs.com/)
+packages and other files.
+
+If you utilize a custom preprocessor, note that this feature requires the latest
```

Comment:
Is there a good place to call out what the error would look like if you didn't use the latest preprocessor? or Webpack at all?

Reply:
Added it right after this line. Addressed in https://github.com/cypress-io/cypress-documentation/pull/4789/commits/590ce263e36bf9c2a5a1014fcad73cb699ec33b6